--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 PREFIX = /usr/local
 
 CHROMIUM_SUFFIX  =
-CHROMIUM_NAME    = chromium$(CHROMIUM_SUFFIX)
+CHROMIUM_NAME    = inox$(CHROMIUM_SUFFIX)
 CHROMIUM_BINARY  = /usr/lib/$(CHROMIUM_NAME)/$(CHROMIUM_NAME)
 CHROMIUM_VENDOR  = $(shell . /etc/os-release; echo $$NAME)
 PEPPER_FLASH_DIR = /usr/lib/PepperFlash
--- a/launcher.c
+++ b/launcher.c
@@ -85,7 +85,7 @@
   fprintf(
       stderr,
       "\n"
-      "Chromium launcher %s -- for Chromium help, see %s(1)\n"
+      "Inox launcher %s -- for Inox help, see %s(1)\n"
       "\n"
       "Custom flags are read from the following file:\n\n"
       "  %s\n\n"
