--- a/help.go
+++ b/help.go
@@ -9,7 +9,7 @@
 	userFlags := fmt.Sprintf("%q", getUserFlags())
 
 	fmt.Println(`
-Chromium launcher ` + launcherVersion + ` -- for Chromium help, see ` + chromiumName + `(1)
+Inox launcher ` + launcherVersion + ` -- for Inox help, see ` + chromiumName + `(1)
 
 Custom flags are read from the following file:
 
--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@
 PREFIX = /usr/local
 
 CHROMIUM_SUFFIX  =
-CHROMIUM_NAME    = chromium$(CHROMIUM_SUFFIX)
+CHROMIUM_NAME    = inox$(CHROMIUM_SUFFIX)
 CHROMIUM_BIN     = /usr/lib/$(CHROMIUM_NAME)/$(CHROMIUM_NAME)
 PEPPER_FLASH_DIR = /usr/lib/PepperFlash
 
