--- a/build/config/compiler/compiler.gni
+++ b/build/config/compiler/compiler.gni
@@ -77,8 +77,7 @@ assert(!is_cfi || use_thin_lto, "CFI req
 # For unofficial (e.g. development) builds and non-Chrome branded (e.g. Cronet
 # which doesn't use Crashpad, crbug.com/479283) builds it's useful to be able
 # to unwind at runtime.
-exclude_unwind_tables = (is_chrome_branded && is_official_build) ||
-                        (is_chromecast && !is_cast_desktop_build && !is_debug)
+exclude_unwind_tables = true
 
 # If true, optimize for size. Does not affect windows builds.
 # Linux & Mac favor speed over size.
